src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,31 @@
+package duke.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+    protected LocalDateTime from;
+    protected LocalDateTime to;
```

Comment:
from/to could be named startTime/endTime or something that identifies the variables have to do with DateTime